### DISCUSSION TITLE
After reddit api changes how are you people accessing reddit?

### DISCUSSION THREAD

**Discussion by bheeshmpita** [spikecodes/libreddit#846] [Q&A]
Till the proper solution emerges, i am using redreader on waydroid to search for reddit posts.
What are your solutions for now?

**Comment by floptical** [comment]
Running my own private libreddit instance works just fine right now, I don't know what all the hub bub is about. This project still perfectly suits my needs.

**Reply by bheeshmpita** [reply]
can you please explain how it is done or a share a guide for beginners. I dont know anything about self-hosting.

**Reply by floptical** [reply]
If you don't know docker or self-hosting I can't walk you through all the pre-requisite knowledge required for self-hosting libreddit in a single comment (nor want to). You'll need to put in the hard work and do some learning. Plenty of good docker guides out there.

Once you understand Docker, libreddit's README has perfectly simple instructions for running it under the Installation --> Docker section.

**Comment by ghost** [comment]
Many users rotated public instances not to avoid ads, but to avoid having their browsing data + IP captured together and catalogued for later use. Self-hosting - even on a VPS - means all requests still flow from one easily identifiable IP to reddit, defeating a significant selling point of the project. Libreddit still has a better UI though 😄

**Comment by apkzarcom** [comment]
[https://apkzar.net](https://apkzar.net
)[https://apkyolo.com](https://apkyolo.com)

**Comment by avincent98144** [comment]
does not load
